benchmarking/runs/priority_algorithm/default_parameters/runtimes.py

```diff
@@ -0,0 +1,150 @@
+                    continue
+
+                student_provider = MockStudentProvider(
+                    settings=MockStudentProviderSettings(
+                        number_of_students=class_size,
+                        friend_distribution="cluster",
+                        number_of_friends=3,
+                        attribute_ranges={
+                            ScenarioAttribute.GENDER.value: [
+                                (Gender.FEMALE, 0.5),
+                                (Gender.MALE, 0.5),
+                            ],
+                            ScenarioAttribute.GPA.value: [
+                                (Gpa.A, 0.5),
+                                (Gpa.B, 0.5),
+                            ],
+                        },
+                    )
+                )
+
+                for scenario_name, scenario in scenarios.items():
+                    artifact = SimulationSet(
+                        settings=SimulationSettings(
+                            num_teams=class_size // team_size,
+                            scenario=scenario,
+                            student_provider=student_provider,
+                            cache_key=f"priority_algorithm/default_parameters/runtimes/{scenario_name}/class_size_{class_size}",
+                        ),
+                        algorithm_set={
+                            AlgorithmType.PRIORITY: [
+                                PriorityAlgorithmConfig(
+                                    MAX_KEEP=30,
+                                    MAX_SPREAD=100,
+                                    MAX_ITERATE=250 * multiplier,
+                                    MAX_TIME=1000000,
+                                    name=f"config_{250 * multiplier}-100-30",
+                                )
+                                if "social" in scenario_name
+                                else PriorityAlgorithmConfig(
+                                    MAX_KEEP=15,
+                                    MAX_SPREAD=30,
+                                    MAX_ITERATE=30 * multiplier,
+                                    MAX_TIME=1000000,
+                                    name=f"config_{30 * multiplier}-30-15",
+                                ),
+                            ],
+                        },
+                    ).run(num_runs=num_trials)
+
+                    metrics = {
+                        "PrioritySatisfaction": PrioritySatisfaction(
+                            goals_to_priorities(scenario.goals),
+                            False,
+                        ),
+                        "AverageSocialSatisfaction": AverageSocialSatisfaction(
+                            metric_function=is_happy_team_1hp_friend,
+                        ),
+                    }
+                    insight_output = Insight.get_output_set(
+                        artifact, list(metrics.values())
+                    )
+                    print(
```

Comment:
Spooky prints, did we start doing this in runs generally?

Reply:
This run only produced runtimes, no graphs, so printing is the simplest way of getting the data here